src/hooks/useErrorCatalogApi/useGetErrorApi.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Configuration,
+  ErrorCatalogApi,
+  InlineResponse2003,
+} from "@openapi/generated";
+import { useState } from "react";
+
+export function useGetErrorApi(
+  accessToken: string,
+  basePath: string
+): {
```

Comment:
I think it should be useful to manage this with our SDK at some point. I've opened [MGDOBR-669](https://issues.redhat.com/browse/MGDOBR-669) to track this activity.